src/arch/x86_64/kernel/processor.rs

```diff
@@ -878,14 +878,18 @@ fn print_cpu_information() {
 }*/
 
 pub fn generate_random_number() -> Option<u32> {
```

Comment:
Is generating u32 values fine? I did not check the interfaces, but how are 64bit random values generated?

Reply:
I added also support for 64 numbers (see latest changes)